slides/04-networking-in-riot/networking-in-riot.md

```diff
@@ -149,14 +150,25 @@ class: center, middle
 
 ---
 
+## IEEE802154 Hardware Access Layer
+
+* Hardware Abstraction Layer (HAL) for radios compliant with the IEEE802.15.4 standard
+
+* Provides a well defined hardware access that allows implementing  agnostic PHY and MAC layers on top
```

Comment:
```suggestion
* Provides a well defined hardware access that allows implementing agnostic PHY and MAC layers on top
```

Reply:
Switched to:

![image](https://user-images.githubusercontent.com/23060007/122400991-7cb40900-cf7c-11eb-89d7-22150f8df5f5.png)

Reply:
Added comparison in previous slide 
![image](https://user-images.githubusercontent.com/23060007/122401050-89d0f800-cf7c-11eb-9438-20f6fd7c6e54.png)